filament/src/details/SkinningBuffer.cpp

```diff
@@ -173,7 +173,7 @@ static inline size_t getSkinningBufferWidth(size_t pairCount) noexcept {
 }
 
 static inline size_t getSkinningBufferHeight(size_t pairCount) noexcept {
-    return (pairCount + MAX_SKINNING_BUFFER_WIDTH) / MAX_SKINNING_BUFFER_WIDTH;
+    return pairCount / MAX_SKINNING_BUFFER_WIDTH + 1;
```

Comment:
I think this should be `(pairCount + MAX_SKINNING_BUFFER_WIDTH - 1) / MAX_SKINNING_BUFFER_WIDTH`, so that you don't get an extra one when it's exactly a multiple of `MAX_SKINNING_BUFFER_WIDTH`

Reply:
You are right, nice solution, thank you